ftw/linkchecker/command/checking_links.py

```diff
@@ -218,23 +215,29 @@ def send_mail_with_excel_report_attached(email_address, path_to_report,
     Friendly regards,\n
     your 4teamwork linkcheck reporter\n\n\n
     '''.format(total_time_fetching_external)
-    portal = api.content.get(site_info['path'])
+    plone_site_path = '/'.join(plone_site_obj.getPhysicalPath())
+    portal = api.content.get(plone_site_path)
     report_mailer_instance = report_mailer.MailSender(portal)
     report_mailer_instance.send_feedback(
         email_subject, email_message, email_address, path_to_report)
 
 
 def main(app, *args):
-    plone_sites_information = get_plone_sites_information(app)
-    for site_info in plone_sites_information:
-        email_address = 'hugo.boss@4teamwork.ch'
+    plone_site_objs = get_plone_sites_information(app)
 
-        setup_plone(app, site_info)
+    path_site_administrator_emails = os.path.join(
+        dirname(dirname(abspath(__file__))), 'site_administrator_emails.json')
```

Comment:
As already mentioned... use a different path

Reply:
I'll address this in issue #29